### PR TITLE
BIP-XXX: Enable NFTE / wsteth-Boosted Aave v3 WETH-BPT Gauge (Arbitrum)

### PR DESCRIPTION
**Pull request by westonnelson**
- Enable NFTE / wsteth-Boosted Aave v3 WETH-BPT Gauge on Arbitrum 
- Tenderly Simulation: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/7c19e2f5-3586-4da1-bb75-e30b067a2f3e
- Pool deployed: [0xD0dC20e6342DB2dE82692B8Dc842301ff9121805](https://app.balancer.fi/#/arbitrum/pool/0xd0dc20e6342db2de82692b8dc842301ff9121805000200000000000000000454)
Gauge deployed: [0xd8699C91E6A4Ca2Dd819607C0ae93264FB2a441d](https://etherscan.io/address/0xd8699C91E6A4Ca2Dd819607C0ae93264FB2a441d#code)

### CODE REVIEW

**Comment by westonnelson** [comment]
Looks like [BIP-349] was just used upon reviewing Discord...have updated the PR to 350 and forum title to 350 but have not changed the .json file. Please let me know if need to do anything, thank you.

**Comment by Xeonus** [comment]
BIP file-name needs to potentially be renamed to prevent clashing with other BIP.json files.

File: BIPs/2023-W26/BIP-349.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (Tritium-VLK) [inline BIPs/2023-W26/BIP-349.json:1]: Files are not put in weekly directories or given BIP numbers until they go to snapshot. 

Please retitle your PR to start with BIP-XXX and rename the file to be called something other than BIP-###, or we will have to close it in the next day or two to not confuse our weekly governance process.

Please put it in the BIPs/00proposed directory.  The file should not be named BIP-XXX and put into a weekly directory until it has been posted to snapshot.  This is done by someone who has over 200,000 veBAL in delegation: https://docs.balancer.fi/concepts/governance/process.html#step-3-develop-and-validate-transaction-pull-request

At the moment this PR is kind of in the way(or can not be merged).

@westonnelson I would request that you close this PR, and reopen it once you have someone with the 200k veBAL to post a snapshot on a Thursday.

When so please follow the instructions above and place the file in the 00proposed directory.

Further our payload report is currently failing on your file. 

Lastly... Your forum post talks about a number of contracts and code, some of which seems quite advanced.   There are no links to any security reviews or audits.  There's also people saying they have had bad experiences engaging in your projects in the past, which go unanswered.

I think you will have an easier time finding someone with 200k veBAL to post this proposal if you spend a bit of time working on your forum post and demonstrating that your project has been reviewed, is safe, somewhat permissionless/etc.

I suggest you close this BIP for now, get the rest of your gauge proposal in a good place, and come back with a PR once other questions have been answered/sorted out in the forum.

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

**Comment by Tritium-VLK** [comment]
1: please rename this file so that
a: It doesn't have a bip number (bip-xxx or named after what it does)
b: lives in 00proposed until someone with 200k veBAL gives it a BIP number and moves it towards snapshot

2: Your forum post does not have a risk section and a lot of comments from the community about risk based concerns.  Please include a risk section in your Forum Post that includes links to any formal reviews/audits and talks about any extra permissions that exist in your system, who has them, and how everything is safeguarded against foul play.

3:  In general opening the PR happens a few days before snapshot, and is done by someone who is posting the snapshot.  This does not seem ready right now for a number of reasons, I suggest you close this PR and re-open it when things are more ready.

**Comment by Tritium-VLK** [comment]
@westonnelson thank you.

**Comment by westonnelson** [comment]
> 1: [X}
> 2: [X]
> 3: [X]

Thank you for the guidance!

**Pull request closed without merging**